web/concrete/src/Permission/Key/Key.php

```diff
@@ -502,4 +502,10 @@ public static function exportTranslations()
 
         return $translations;
     }
+
+    public static function clearCache()
+    {
+        $cache = Core::make('cache/expensive');
```

Comment:
and what happens to all the packages out there? If a package developer adds a new permission key in his package controller things will fall apart.. He'd have to be aware to clear the cache, feels very error-prone to me.